pages/_app/index.jsx

```diff
@@ -5,28 +5,25 @@ import { withRouter } from 'next/router'
 import '@oacore/design/lib/index.css'
 
 import Route from './route'
-import { initializeData, GlobalProvider } from '../../store'
+import { initStore, GlobalProvider } from '../../store'
 
 import Application from 'components/application'
 
 class App extends NextApp {
   static async getInitialProps({ Component, ctx }) {
-    const initialStoreData = initializeData()
+    const store = initStore()
     let pageProps = {}
-
     // Provide the store to getInitialProps of pages
-    if (Component.getInitialProps) {
-      pageProps =
-        (await Component.getInitialProps({ ...ctx, initialStoreData })) || {}
-    }
+    if (Component.getInitialProps)
+      pageProps = (await Component.getInitialProps({ ...ctx, store })) || {}
 
     return {
       pageProps,
-      initialStoreData,
+      store,
     }
   }
 
-  handleNavigation = event => {
+  handleNavigation = async event => {
```

Comment:
I did not know about. Sorry and thank you!

Reply:
It is not needed. I forgot to remove it. I was just testing something.